src/lib/xiaomi.ts

```diff
@@ -1381,6 +1375,40 @@ export const xiaomiModernExtend = {
         attribute: 'presentValue',
         ...args,
     }),
+    aqaraVoc: (args?: Partial<modernExtend.NumericArgs>) => modernExtend.numeric({
+        name: 'voc',
+        cluster: 'genAnalogInput',
+        attribute: 'presentValue',
+        configureReporting: {minimumReportInterval: 10, maximumReportInterval: constants.repInterval.HOUR, reportableChange: 5},
```

Comment:
Let's remove `maximumReportInterval` and use the default from modernExtend instead (`repInterval.MAX`))

Reply:
I actually copied those from what is currently in the configure() function and those seemed like a decent value. (which I forgot to cleanup, rebasing that commit to also clean this up)
```
const payload = reporting.payload('presentValue', 10, constants.repInterval.HOUR, 5);
```

Dropping this sets it to `65000` which seems a bit high, I would not set it longer than 1x 3h.

Reply:
> Dropping this sets it to 65000 which seems a bit high, I would not set it longer than 1x 3h.

I see, then this is fine